include/staff/ticket-tasks.inc.php

```diff
@@ -40,12 +40,17 @@ class="green button action-button ticket-task-action"
             print __('Add New Task'); ?></a>
     <?php
     }
+    foreach ($tasks as $task)
```

Comment:
To me it looks like the '$taskStatus` will be set to the status of the last task - is that the intended?

Reply:
No, it is appending to one string, so it would have all of the values. Then in tasks-actions.tmpl.php, we determine which options belong in the list (Reopen/Close) based on that status string. Let me know if you would like this implemented a different way.